webapp/store/models.py

```diff
@@ -10,6 +10,25 @@
 cs = CharmStore("https://api.jujucharms.com/v5")
 
 
+def search_entities(query):
+    includes = [
+        'charm-metadata',
+        'bundle-metadata',
+        'owner',
+        'bundle-unit-count',
+        'stats',
```

Comment:
I don't think we need stats anymore unless we're going to weight them differently than they are returned from the API.